tests/common/plugins/loganalyzer/loganalyzer.py

```diff
@@ -17,15 +17,14 @@
 COMMON_EXPECT = join(split(__file__)[0], "loganalyzer_common_expect.txt")
 SYSLOG_TMP_FOLDER = "/tmp/syslog"
 
-
 class LogAnalyzerError(Exception):
     """Raised when loganalyzer found matches during analysis phase."""
     def __repr__(self):
         return pprint.pformat(self.message)
 
 
 class LogAnalyzer:
-    def __init__(self, ansible_host, marker_prefix, dut_run_dir="/tmp", start_marker=None):
+    def __init__(self, ansible_host, marker_prefix, dut_run_dir="/tmp", start_marker=None, additional_files=[], additional_start_str=[]):
```

Comment:
Since `additional_files` and `additional_start_str` are interdependent, we may want to use dict.

For example, caller will send
```Python
additional_files: { '/var/log/swss/sairedis.rec':  'recording on: /var/log/swss/sairedis.rec' }
loganalyzer = LogAnalyzer(ansible_host=duthost, marker_prefix="test_advanced_reboot", additional_files=additional_files)
```

I think in that case, you don't have to have the below check either:
`if self.additional_files and self.additional_start_str and len(self.additional_files) != len(self.additional_start_str)`

Reply:
Updated as suggested.